content/en/docs/chart_template_guide/control_structures.md

```diff
@@ -164,7 +164,7 @@ data:
 ```
 
 Notice that we received a few empty lines in our YAML. Why? When the template
-engine runs, it _removes_ the contents inside of `{{` end `}}`, but it leaves
+engine runs, it _removes_ the contents inside of `{{` and `}}`, but it leaves
```

Comment:
This should be `end`

Reply:
Can you explain why? The space appears to be left not as a result of `{{ end }}` but the general contents inside the braces.